sources/packages/web/src/assets/css/base.scss

```diff
@@ -1,12 +1,12 @@
 @import "./global-style-variables.scss";
 html * {
-  font-family: "BCSans", "Noto Sans", Verdana, Arial, sans-serif, "primeicons";
+  font-family: "BCSans", "Noto Sans", Verdana, Arial, sans-serif;
   font-style: $font-style-normal;
   line-height: 20px;
 }
 // todo: when all v-navigation-drawer is updated, remove this class.
 .body-background {
-  background: $color-background;
+  background: $default-background;
```

Comment:
The TODO says that this could be removed "v-navigation-drawer is updated". Is it not updated yet?

Reply:
it meant in after all ui refationing